src/lib/__tests__/cache.test.js

```diff
@@ -38,6 +38,17 @@ describe("Cache", () => {
       })
     })
 
+    describe("#delete", () => {
+      beforeEach(() => cache.set("get_foo", { bar: "baz" }))
+
+      it("deletes the data", () => {
+        cache.delete("get_foo")
+        return cache.get("get_foo").catch(e => {
```

Comment:
This test doesn’t seem to be fully correct. If the promise returned by `get(…)` were to resolve then you’d be returning a resolved promise to the test case. This means the test would pass, even if it’s _not_ the correct behaviour.

You can do the following to tell the test case that at least 1 assertion is expected:

```js
expect.assertions(1)
return cache.get("get_foo").catch(e => { … })
```